src/robot/running/model.py

```diff
@@ -175,6 +176,34 @@ def run(self, context, run=True, templated=False):
                 raise ReturnFromKeyword(self.values)
 
 
+@Body.register
+class Continue(model.Return):
```

Comment:
There should be `Continue` and `Break` classes in the `model` package that are extended here. They are also needed in the `result` package.

Reply:
added in result pkg but need you guidance a not able to understand this pkg fully.